mne_connectivity/spectral/epochs_multivariate.py

```diff
@@ -31,8 +33,14 @@ def _check_rank_input(rank, data, indices):
             if "copy" in inspect.getfullargspec(data.get_data).kwonlyargs:
                 kwargs["copy"] = False
             data_arr = data.get_data(**kwargs)
-        else:
-            data_arr = data
+        elif isinstance(data, (EpochsSpectrum, EpochsSpectrumArray)):
+            # Spectrum objs will drop bad channels, so specify picking all channels
+            data_arr = data.get_data(picks=np.arange(data.info["nchan"]))
+            # Convert to power (and aggregate over tapers) before computing rank
+            if "taper" in data._dims:
+                data_arr = _psd_from_mt(data_arr, data.weights)
+            else:
+                data_arr = (data_arr * data_arr.conj()).real
```

Comment:
Also forgot to aggr. over tapers before checking the rank (otherwise we have a tapers dim in the singular values, and I'm not sure how to handle that).

Converting to power isn't explicitly necessary I guess, but I at least know it's a proper way to handle the tapers. And if we're doing that for multitaper coeffs, should keep it consistent and convert to power for Fourier/Welch as well. Also they're just simple matmuls, so not a huge overhead in this additional step.